src/rpc/methods/f3.rs

```diff
@@ -430,15 +435,31 @@ impl RpcMethod<0> for GetParticipatingMinerIDs {
     type Params = ();
     type Ok = Vec<u64>;
 
-    async fn handle(ctx: Ctx<impl Blockstore>, _: Self::Params) -> Result<Self::Ok, ServerError> {
-        match ctx.chain_config().network {
-            NetworkChain::Devnet(_) => {
-                // For now, just hard code the devnet miner for testing
-                let shared_miner_addr = Address::from_str("t01000")?;
-                Ok(vec![shared_miner_addr.id()?])
+    async fn handle(_: Ctx<impl Blockstore>, _: Self::Params) -> Result<Self::Ok, ServerError> {
+        const ENV_KEY: &str = "FOREST_F3_PERMENANT_PARTICIPATING_MINER_ADDRESSES";
+        let mut ids: Vec<u64> = F3_PARTICIPANT_LEASES.read().keys().copied().collect();
+        if let Ok(permenant_addrs) = std::env::var(ENV_KEY) {
```

Comment:
Would it be worth covering it with unit tests? There's quite a bit of loops and conditionals there to make changes there potentially faulty.

Reply:
Tests added.